### ISSUE TITLE
No transition when mwc-menu is closed

### ISSUE THREAD

**Issue by ForestKatsch** [material-components/material-components-web-components#1975]
**Describe the bug**
There is no animated transition when `mwc-menu` is closed (the opening transition works as expected.)

**To Reproduce**
1. Go to https://material-components.github.io/material-components-web-components/demos/menu/
2. Click on any menu button.
3. Click on a menu item.
4. The menu instantly disappears on mouseup.

**Expected behavior**
The menu should transition out, similar to how it transitions in.

**Screenshots**

**Browser Version (please complete the following information):**
 - OS: macOS 11
 - Browser: Chrome, Safari

**Additional context**
N/A

**Comment by asyncLiz** [comment]
Thanks for the issue!

I've confirmed that the underlying `mdc-menu-surface--animating-closed` opacity fade transition does not appear when slowing animations on the page down to 10% with the developer tools. If I remove `display: none` from the `mdc-menu-surface` element, then the closing animation appears correctly.

I've also confirmed this in our upstream library and internal catalog. Since this is an upstream issue, I'll move this issue over to the MDC library to fix there.
